sonar-ws/src/main/protobuf/ws-notifications.proto

```diff
@@ -35,6 +35,7 @@ message ListResponse {
 message Notification {
   optional string channel = 1;
   optional string type = 2;
+  optional string organization = 5;
```

Comment:
but the error doesn't last past the compilation, so...

I think it's a tradeoff between backward compatibility and field sorting that functionally makes sense